examples/tutorials/nb_python/rigid_object_tutorial.py

```diff
@@ -148,16 +182,29 @@ def simulate(sim, dt=1.0, get_frames=True):
 
 
 # [/setup]
-
-# This is wrapped such that it can be added to a unit test
-def main(make_video=True, show_video=True):
+if __name__ == "__main__":
+    import argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--no-show-video', dest='show_video', action='store_false')
```

Comment:
This wouldn't pass black tho -- and the CI has `examples/tutorials/(colabs|nb_python)` in the ignores.

Reply:
It passes black with the exception of that though and I manually run black on all the files before committing them anyway so it shouldn't be a big issue, right? I have an open issue with JuPyText to see if we can't get it resolved.

Reply:
> Are any files importing/consuming vis_utils.py yet?

Yeah RigidObjects uses the display video function currently. I need to get factor out the video display one though.

Reply:
Yeah, agreed on the exclude for the notebooks.  Manually having to run formatters is sad :(